lib/usdUfe/ufe/UsdUndoInsertChildCommand.cpp

```diff
@@ -18,8 +18,11 @@
 
 #include "private/UfeNotifGuard.h"
 
+#include <mayaUsd/utils/layers.h>
```

Comment:
Sorry you cannot do this. UsdUfe is an independent library that is linked into mayaUsd. So it is now allowed to include any mayaUsd headers. I think you just meant to include `usdUfe/utils/layers.h`

Reply:
Yeah, this was written before your PR :)